tests/InputFiles/ScalarWave/Input1DPeriodic.yaml

```diff
@@ -0,0 +1,30 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+# Executable: EvolveScalarWave1D
+# Check: execute
+AnalyticSolution:
+  WaveVector: [1.0]
+  Center: [0.0]
+  Profile:
+    Sinusoid:
+      Amplitude: 1.0
+      Wavenumber: 1.0
+      Phase: 0.0
+
+InitialTime: 0.0
+FinalTime: 0.1
+DeltaT: 0.01
+
+DomainCreator:
+  Interval:
+    LowerBound: [0.0]
+    UpperBound: [6.283185307179586]
+    IsPeriodicIn: [true]
+    InitialRefinement: [2]
+    InitialGridPoints: [7]
+
+TimeStepper:
+  AdamsBashforthN:
+    TargetOrder: 3
+
+NumericalFlux:
```

Comment:
Why does this work? I would've thought no numerical flux being specified would be an error.

Reply:
NumericalFlux -> NumericalFluxParameters